independent-projects/qute/core/src/test/java/io/quarkus/qute/EscaperTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.qute;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.quarkus.qute.TemplateNode.Origin;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+
+/**
+ * 
+ */
+public class EscaperTest {
+
+    @Test
+    public void testEscaping() throws IOException {
+        Escaper escaper = Escaper.builder().add('a', "aaa").build();
+        assertEquals("aaa", escaper.escape("a"));
+        assertEquals("b", escaper.escape("b"));
+
+        Escaper html = Escaper.builder().add('"', "&quot;").add('\'', "&#39;")
+                .add('&', "&amp;").add('<', "&lt;").add('>', "&gt;").build();
+        assertEquals("&lt;strong&gt;Čolek&lt;/strong&gt;", html.escape("<strong>Čolek</strong>"));
+        assertEquals("&lt;a&gt;&amp;link&quot;&#39;&lt;/a&gt;", html.escape("<a>&link\"'</a>"));
+    }
+
+    @Test
+    public void testRawStringRevolver() {
```

Comment:
```suggestion
    public void testRawStringResolver() {
```

Reply:
I may be blind but what's the suggested change?

Reply:
Revolver != resolver :smile:

Reply:
Ok, it's confirmed. I'm blind...

Reply:
Most beautiful typo I've seen so far :)

Reply:
I'm very good at typos!